config/development.json

```diff
@@ -13,6 +13,7 @@
     "warningBanner": false,
     "helpWithFees": true,
     "pcq":true,
+    "AntennawebChat": false,
```

Comment:
is this related to 9081 change?

Reply:
it's not. this is to keep local dev loads pages quickly.